src/actors.c

```diff
@@ -20,6 +20,26 @@
 #include <sounds.h>
 #include "audio/external.h"
 #include "common_textures.h"
+#include "courses/battle/big_donut/course_data.inc.h"
+#include "courses/battle/block_fort/course_data.inc.h"
+#include "courses/battle/double_deck/course_data.inc.h"
+#include "courses/battle/skyscraper/course_data.inc.h"
+#include "courses/mushroom_cup/kalimari_desert/course_data.inc.h"
+#include "courses/mushroom_cup/koopa_troopa_beach/course_data.inc.h"
+#include "courses/mushroom_cup/luigi_raceway/course_data.inc.h"
+#include "courses/mushroom_cup/moo_moo_farm/course_data.inc.h"
+#include "courses/flower_cup/choco_mountain/course_data.inc.h"
+#include "courses/flower_cup/frappe_snowland/course_data.inc.h"
+#include "courses/flower_cup/mario_raceway/course_data.inc.h"
+#include "courses/flower_cup/toads_turnpike/course_data.inc.h"
+#include "courses/star_cup/bowsers_castle/course_data.inc.h"
+#include "courses/star_cup/royal_raceway/course_data.inc.h"
+#include "courses/star_cup/sherbet_land/course_data.inc.h"
+#include "courses/star_cup/wario_stadium/course_data.inc.h"
+#include "courses/special_cup/banshee_boardwalk/course_data.inc.h"
+#include "courses/special_cup/dks_jungle_parkway/course_data.inc.h"
+#include "courses/special_cup/rainbow_road/course_data.inc.h"
+#include "courses/special_cup/yoshi_valley/course_data.inc.h"
```

Comment:
I'm thinking we should have a course_data.h that contains the above. So you don't have to type every single course like this. (that can be in a future PR if someone decides to do so, and if not, that's fine too).

I don't see a reason for only including like two courses versus just always including all of them. I suppose naming conflicts could happen but I think that would be unlikely given the naming scheme. Plus probably better to catch those sooner rather than later.

Reply:
I did consider making a top level `all_course_data.h` or something to that effect at the top level of the `courses` directory. I can do that if you want.

Reply:
I think the all_course_data.h would be a great idea. Feel free to leave that for a future PR if you wish.

Reply:
I went ahead and did it. Its a small thing anyway.